### PR TITLE
docs: Issue templates

### PR DESCRIPTION
**Pull request by SashoStoichkov**
Issue #8

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.md
Comment (nhcarrigan) [inline .github/ISSUE_TEMPLATE/bug.md:5]:
```suggestion
labels: ''
```

File: .github/ISSUE_TEMPLATE/docs.md
Comment (nhcarrigan) [inline .github/ISSUE_TEMPLATE/docs.md:5]:
```suggestion
labels: ''
```

File: .github/ISSUE_TEMPLATE/ideas.md
Comment (nhcarrigan) [inline .github/ISSUE_TEMPLATE/ideas.md:5]:
```suggestion
labels: ''
```

**Review by nhcarrigan — CHANGES_REQUESTED** [review]
I think we should hold off on auto-labelling until we have set up the label system we choose to use.

Thoughts?

File: .github/ISSUE_TEMPLATE/bug.md
Comment (Vyvy-vi) [inline .github/ISSUE_TEMPLATE/bug.md:22]: I am not sure if we should include the link to the discord
(this is a project for the client and maybe we should not do this?)

*maybe we can get a second opinion on this from someone ?

File: .github/ISSUE_TEMPLATE/ideas.md
Comment (Vyvy-vi) [inline .github/ISSUE_TEMPLATE/ideas.md:3]: Maybe we can use "for the project?"
I thin this isn't limited to a bot(and ideas may vary and be about stuff not related to the bot)

File: .github/ISSUE_TEMPLATE/docs.md
Comment (Vyvy-vi) [inline .github/ISSUE_TEMPLATE/docs.md:3]: Maybe we could also add- "for documentation requests/ideas/issues?"

File: .github/ISSUE_TEMPLATE/bug.md
Comment (Vyvy-vi) [inline .github/ISSUE_TEMPLATE/bug.md:3]: I'm not sure if the project is _only_ limited to the bot... Maybe there's some bug in an implementation of the API used rather than the bot?

File: .github/ISSUE_TEMPLATE/bug.md
Comment (eddiejaoude) [reply .github/ISSUE_TEMPLATE/bug.md:22]: hmm interesting, I think this is ok for now, we can always remove later

Comment (SashoStoichkov) [reply .github/ISSUE_TEMPLATE/bug.md:22]: Okay - I'll leave it there for now 👍🏻

File: .github/ISSUE_TEMPLATE/ideas.md
Comment (SashoStoichkov) [reply .github/ISSUE_TEMPLATE/ideas.md:3]: Okay - I'll change that

File: .github/ISSUE_TEMPLATE/docs.md
Comment (SashoStoichkov) [reply .github/ISSUE_TEMPLATE/docs.md:3]: I am not sure I quite understand. Can you clarify please, @Vyvy-vi?

**Comment by eddiejaoude** [comment]
We now have labels 🎉 to find out what they are or if you need to add more https://github.com/eddiejaoude/stargate/blob/main/docs/labels.md

File: .github/ISSUE_TEMPLATE/docs.md
Comment (Vyvy-vi) [reply .github/ISSUE_TEMPLATE/docs.md:3]: oh wupps

Comment (Vyvy-vi) [reply .github/ISSUE_TEMPLATE/docs.md:3]: I read the filename wrong 😓

File: .github/ISSUE_TEMPLATE/bug.md
Comment (Vyvy-vi) [reply .github/ISSUE_TEMPLATE/bug.md:22]: :+1:

**Review by Vyvy-vi — APPROVED** [review]
LGTM-!
But since, Eddie stated it some labels could be added(but I guess those can come in a later PR 🤓 )

**Review by AllanRegush — APPROVED** [review]
Looks Good 👍 This already adds a bunch of value.

**Comment by eddiejaoude** [comment]
Looks good 👍 - yep labels can be added later as mentioned by @Vyvy-vi and already adds value as mentioned by @AllanRegush - great team work 💪

**Comment by SashoStoichkov** [comment]
Thank you everyone 👍🏻

**Pull request merged**
